app/views/dashboard/_host_reports_status_chart_widget.html.erb

```diff
@@ -3,9 +3,9 @@
 %>
 <h4 class="header">
   <% if format %>
-    <%= _('%s Host Reports Chart') % format %>
+    <%= _('Latest %s Host Reports Chart') % format %>
   <% else %>
-    <%= _('All Host Reports Chart') %>
+    <%= _('Latest Host Reports Chart') %>
   <% end %>
```

Comment:
I do not remember exactly the feedback from Maria on our meeting but we agreed to name this Latest Host Reports.